### PR TITLE
Assert docs and change deps

### PR DESCRIPTION
**Pull request by murilo-cunha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,21 @@ $ databooks fix [OPTIONS] PATHS...
...
+comply with the desired metadata values. This advanced feature allows users to enforce
+cell tags, sequential cell execution, maximum number of cells, among many other things!
+
+Check out our [docs](https://databooks.dev/latest/usage/overview/#databooks-assert) for more!
+
+```console
+$ databooks assert [OPTIONS] PATHS...
+```
+
+![databooks fix demo](https://raw.githubusercontent.com/datarootsio/databooks/main/docs/images/databooks-assert.gif)
```
Comment (tleers) [inline README.md:82]: This image is a bit difficult to read for me - not a huge issue but something that could be improved

File: docs/usage/overview.md [hunk 2]
```diff
@@ -1,4 +1,229 @@
...
+
+```bash
+databooks meta path/to/notebooks
+```
+
+With that, for each notebook in the path, by default:
+
+- It will remove execution counts
+- It **won't** remove cell outputs
+- It will remove metadata from all cells (like cell tags or ids)
```
Comment (tleers) [inline docs/usage/overview.md:30]: (such as cell tags or ids)

File: README.md (hunk 1 above)
Comment (murilo-cunha) [reply README.md]: Changed the image name fyi

File: docs/usage/overview.md [hunk 3]
```diff
@@ -1,4 +1,229 @@
...
+
+This can be used, for example, to make sure that the notebook cells were executed in
+order. Or that we have `markdown` cells, or to set a maximum number of cells for each
+notebook.
+
+Evidently, there are some limitations to the expressions that a user can pass.
+
+**Variables in scope:**
+
+All the variables in scope are a subclass of Pydantic models. Therefore, you can use them
```
Comment (tleers) [inline docs/usage/overview.md:104]: This is a lot of stuff to throw at an average data scientist in an overview md IMO. 
Variables in scope? 
Subclass of Pydantic?
The example doesn't make it super clear to a newb like me either 

```suggestion
All of the variables that you may use in your assert expressions are [subclasses of Pydantic models](link). Therefore, you can use **them** 
```
Specify what "them" is here. Is this related to metadata? To cell metadata? Other metadata?

File: docs/usage/overview.md [hunk 4]
```diff
@@ -1,4 +1,229 @@
...
+as regular python objects (i.e.: to access the cell types, one could write
+`[cell.cell_type for cell in nb.cells]`). For convenience, are available the variables
```
Comment (tleers) [inline docs/usage/overview.md:106]:
```suggestion
`[cell.cell_type for cell in nb.cells]`). For convenience's sake, here is a list of currently supported variables that can be used in assert expressions:
```

File: docs/usage/overview.md [hunk 5]
```diff
@@ -1,4 +1,229 @@
...
+
+These limitations are designed to allow anyone to use `databooks assert` safely. This is
+because we use built-in's [`eval`](https://docs.python.org/3/library/functions.html#eval),
+and [`eval` is really dangerous](https://nedbatchelder.com/blog/201206/eval_really_is_dangerous.html).
+To mitigate that (and for your safety), we actually parse the string and only allow a
+couple of operations to happen. Check out our [tests](https://github.com/datarootsio/databooks/blob/main/tests/test_affirm.py)
+to see what is and isn't allowed and see the [source](https://github.com/datarootsio/databooks/blob/main/databooks/affirm.py)
+to see how that happens!
+
+It's also relevant to mention that to avoid repetitive typing one configure the tool to
```
Comment (tleers) [inline docs/usage/overview.md:158]:
```suggestion
It's also relevant to mention that to avoid repetitive typing you can configure the tool to
```

File: docs/usage/overview.md [hunk 6]
```diff
@@ -1,4 +1,229 @@
...
+omit the source string. Even more powerful is to combine it with [`pre-commit`](https://pre-commit.com/)
```
Comment (tleers) [inline docs/usage/overview.md:159]:
```suggestion
omit the source string. An even more powerful method is to combine it with [`pre-commit`](https://pre-commit.com/)
```

File: docs/usage/overview.md [hunk 7]
```diff
@@ -1,4 +1,229 @@
...
+or CI/CD. Check out the rest of the "Usage" section for more info!
+
+### Recipes
+
+It can be a bit repetitive and tedious to write out expressions to be asserted. Or
+even hard to think of how to express these assertions about notebooks. With that in mind,
+we also include "user recipes". These recipes store some useful expressions to be checked,
+to be used both as shorthand of other expressions and inspiration for you to come up
+with your own recipe! Feel free to submit a PR with your recipe or open an issue if
+you're  having issues coming up with it.
```
Comment (tleers) [inline docs/usage/overview.md:169]:
```suggestion
you're having issues coming up with a recipe for your intended goal.
```

File: docs/usage/cli.md [hunk 8]
```diff
@@ -1,7 +1,9 @@
 # CLI tool
 
-The most straightforward way is to use it in the terminal, whenever desired. That can be
-error-prone and have "dirty" notebooks in your git repo. Check [CLI documentation](../CLI)
-for more information.
+The most straightforward way is to use it in the terminal, whenever desired. It's the
```
Comment (tleers) [inline docs/usage/cli.md:3]:
```suggestion
The most straightforward way to start using databooks is by using the terminal. It's the
```

File: docs/usage/cli.md [hunk 9]
```diff
@@ -1,7 +1,9 @@
...
+default way of running the commands and how you've probably seen `databooks` being used
+before. However, that can be error-prone and have "dirty" notebooks in your git repo.
```
Comment (tleers) [inline docs/usage/cli.md:5]:
```suggestion
before. However, using the terminal can be error-prone and result in "dirty" notebooks in your git repo.
```

File: docs/usage/cli.md [hunk 10]
```diff
@@ -1,7 +1,9 @@
...
+Check [CLI documentation](../CLI) for more information.
 
-A safer alternative is to automate this step, by setting up CI in your repo or a pre-commit hook.
\ No newline at end of file
+A safer alternative is to automate this step, by setting up CI in your repo or
```
Comment (tleers) [inline docs/usage/cli.md:8]:
```suggestion
A safer alternative is to automate your databooks commands, by setting up CI in your repo or
```

**Review by tleers — CHANGES_REQUESTED** [review]

File: docs/usage/overview.md (hunk 3 above)
Comment (murilo-cunha) [reply docs/usage/overview.md:104]: That makes a lot of sense! Thanks for the suggestion!

**Review by tleers — APPROVED** [review]
